lib/index.js

```diff
@@ -33,15 +33,13 @@ var CustomerIO = module.exports = integration('Customer.io')
  */
 
 CustomerIO.prototype.identify = function(identify, fn){
-  var self = this;
-  this.visit(identify, function(){
-    self
-    .put(identify.userId())
-    .auth(self.settings.siteId, self.settings.apiKey)
-    .type('json')
-    .send(mapper.identify(identify))
-    .end(self.handle(fn));
-  });
+  if (!identify.active()) return setImmediate(fn);
```

Comment:
i don't think we should return early here — `msg.active()` returns false in the case when a customer sends a call because they want it recorded (an offline event happened or updating a user profile) but they specifically don't want to update `last_visit` or have the call count towards a session in tools where we can opt out from that. I'm fine with removing `last_visit` altogether in `identify` and `track` if that aligns better with c.io expectations, but it doesn't seem necessary to skip identifies and tracks that signal "this call is not part of an active session". 

thoughts? :)